packages/react/src/components/UIShell/SideNavItems.js

```diff
@@ -19,6 +20,12 @@ const SideNavItems = ({
 }) => {
   const className = cx([`${prefix}--side-nav__items`], customClassName);
   const childrenWithExpandedState = React.Children.map(children, child => {
+    if (
+      typeof child.type === 'string' ||
+      (child.type === SideNavLink && child.props.element === 'a')
```

Comment:
Hi 👋 thank you for another try - Would you try investigating more wrt what components get `isSideNavExpanded` prop as a child of `<SideNavItems>`?

Reply:
Sure, sound good

Reply:
From all the components that have `isSideNavExpanded` property (`SideNavHeader`, `SideNavFooter`, `SideNavLink`, `SideNavMenu`, `UIShell/Link`) only `SideNavLink` and `UIShell/Link` are ones that could potentially pass it to DOM element and produce the warning, I will add another check for UIShell/Link

Reply:
is `element` property used consistently through out the project? If that is the case I could just check if that property value is a string and that would handle more scenarious

Reply:
Not certain, and I imagine it easily gets no longer the case even if it's the case now. Also, such prop doesn't seem to be in use. BTW wondering why you thought of adding such check...?

Reply:
nvm, looks like this is already fixed